app/app.go

```diff
@@ -739,12 +739,41 @@ func New(
 		return app.mm.RunMigrations(ctx, app.configurator, fromVM)
 	})
 
+	gravityPlanName := "v0.8.0-gravity-alpha1"
+	if experimental {
+		app.UpgradeKeeper.SetUpgradeHandler(gravityPlanName, func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+			updatedVM, err := app.mm.RunMigrations(ctx, app.configurator, fromVM)
+			if err != nil {
+				return nil, err
+			}
+			// register new params
+			gravParamStore := app.GetSubspace(gravitytypes.ModuleName)
+			gravParamStore.Set(ctx, gravitytypes.ParamStoreBridgeActive, true)
+			gravParamStore.Set(ctx, gravitytypes.ParamStoreBatchCreationPeriod, uint64(10))
+			gravParamStore.Set(ctx, gravitytypes.ParamStoreBatchMaxElement, uint64(100))
+			gravParamStore.Set(ctx, gravitytypes.ParamStoreObserveEthereumHeightPeriod, uint64(50))
+
+			// set new gravity id
+			gravParams := app.GravityKeeper.GetParams(ctx)
+			gravParams.GravityId = "cronos_gravity_pioneer_v2"
+			app.GravityKeeper.SetParams(ctx, gravParams)
+
+			// Estimate time upgrade take place
+			// 100% is not necessary here because it will be tuned by relayers later on
+			// it is set to georli height at 23th August 2022 3pm JST
+			app.GravityKeeper.MigrateGravityContract(
```

Comment:
for info @lezzokafka , upgrade day should be after 23th August 2022 3pm JST so 24th should be okay